### PR TITLE
Cube fit position - new feature

### PR DESCRIPTION
**Pull request by guysopher**
Align images in Grid Fit to different directions

### CODE REVIEW

**Review by nirnaor — APPROVED** [review]
I tried to avoid mutations in your changes but I think it's better to keep this file consistent

**Review by IzaacAyelin — COMMENTED** [review]
maybe we should add some e2es for the different options

**Comment by guysopher** [comment]
@yhattav @IzaacAyelin can I merge this?

File: packages/gallery/tests/e2e/styleParams/cubeFitPosition.e2e.spec.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  });
+  it('should align to the top', async () => {
+    await driver.navigate({
+      galleryLayout: GALLERY_CONSTS.layout.GRID,
+      cubeType: GALLERY_CONSTS.cubeType.FIT,
+      cubeFitPosition: GALLERY_CONSTS.cubeFitPosition.TOP,
+    });
+    await driver.waitFor.hookToBeVisible('item-container');
+    const page = await driver.grab.elemScreenshot('.pro-gallery');
+    expect(page).toMatchImageSnapshot();
```
Comment (nirnaor) [inline packages/gallery/tests/e2e/styleParams/cubeFitPosition.e2e.spec.js]: @guysopher This entire code is duplicated, makes it impossible to understand the difference between the tests.
Pleaes reuse the duplicated props and the logic parts:
```
    await driver.waitFor.hookToBeVisible('item-container');
    const page = await driver.grab.elemScreenshot('.pro-gallery');
    expect(page).toMatchImageSnapshot();
```

**Review by nirnaor — CHANGES_REQUESTED** [review]

File: packages/gallery/tests/e2e/styleParams/cubeFitPosition.e2e.spec.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+      galleryLayout: GALLERY_CONSTS.layout.GRID,
+      cubeType: GALLERY_CONSTS.cubeType.FIT,
+      cubeFitPosition: GALLERY_CONSTS.cubeFitPosition.LEFT,
+    });
+    await driver.waitFor.hookToBeVisible('item-container');
+    const page = await driver.grab.elemScreenshot('.pro-gallery');
+    expect(page).toMatchImageSnapshot();
+  });
+  it('should align to the right', async () => {
+    await driver.navigate({
```
Comment (nirnaor) [inline packages/gallery/tests/e2e/styleParams/cubeFitPosition.e2e.spec.js]: All tests should look like this:
At the Top level please add:
```
const {TOP, RIGHT, LEFT, BOTTOM} = GALLERY_CONSTS.cubeFitPosition;
```

```
it('should align to the right', async () => {
    await driver.navigate(gridFitWithPosition(RIGHT)});
    
    await expectGalleryToMatchSnapshot()
  });
```

**Review by nirnaor — CHANGES_REQUESTED** [review]

**Comment by IzaacAyelin** [comment]
@nirnaor can this be merged?

**Review by nirnaor — APPROVED** [review]

**Comment by nirnaor** [comment]
> @nirnaor can this be merged?

yes. and sorry for the late response

**Pull request merged**